css/selectors/picture-in-picture.json

```
@@ -23,7 +23,7 @@
             "opera": "mirror",
             "opera_android": "mirror",
             "safari": {
-              "version_added": false
+              "version_added": "≤14.1"
```

https://bugs.webkit.org/show_bug.cgi?id=203493
https://github.com/WebKit/WebKit/commit/99d3116a7c0199ef6ece0f0800765b8d5ef39363

Also, 13.1 is aligned with https://developer.mozilla.org/en-US/docs/Web/API/PictureInPictureWindow#browser_compatibility

```suggestion
              "version_added": "13.1"
```